### ISSUE TITLE
Cannot delete page (win10, Zim 0.73.5 portable)

### ISSUE THREAD

**Issue by shulist** [zim-desktop-wiki/zim-desktop-wiki#1511]
I created a page titled `2168545 Ontario Inc..txt` and it resides in with a list of other pages. It turned out that the leading number was wrong so I created a new page titled with the correct number, eliminated the double period and copied what data was in the first page to the second. 

When I attempt to delete this first page it goes through the motions (Remove Links, Delete File) but still will not delete. Weirdly the text file does not exist for that entry anywhere in the notebook tree.

I thought possibly that the double period was the culprit so attempted to rename the page but even that does not work. There are no backlinks to this page and nothing seems to be unusual about the text in it.

Could someone point me in the direction of how this file can be deleted, or is there a way to rebuild the zim database to correct this?

**Comment by jaap-karssenberg** [comment]
What do you see after delete?  You say it won't delete but the file does
not exist, what do you see that is unexpected?

Op do 24 jun. 2021 15:43 schreef shulist ***@***.***>:

**Comment by shulist** [comment]
OK so I think I solved my problem. It seems that Zim creates folders in the PortableApps `cache `directory. These folders contain two files `index.db` & `state.conf`. These folders are named notebook-[Drive letter]-Zim_... and seem never to be deleted as in my case there were about 12 of them dating back to the time I started using Zim. 

In looking at dates on the folders I managed to remove (temporarily renamed them first) all but two of them as the others were dated from many months ago. I think the offending folder was one named `notebook-X_Zim` this was ultimately recreated upon the restart of Zim. 

After that all was OK. I still don't know why it happened but if it occurs again I will be more diligent in recording the results.

@jaap-karssenberg thanks for your quick response. However is there an issue with Zim creating all of these folders in the `cache` directory and not removing any of them? It also seems to create numerous folders in the  `C:\Users\[Username]\AppData\Local\Temp` and leaves those without removing them later-there were at least 46 folders created by Zim in this folder. They each only contain a `zim.log` file. Is it not possible that this log file be written to the PortableApps `cache` folder?
